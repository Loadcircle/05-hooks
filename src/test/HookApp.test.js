import React from 'react';
const { shallow } = require("enzyme");
const { default: HookApp } = require("../HookApp");

describe('Pruebas en el hook app', ()=>{

    const wrapper = shallow(<HookApp />);

    test('Match component', ()=>{
        expect(wrapper).toMatchSnapshot();
    });

    test('Componen content', ()=>{
        expect(wrapper.find('div').props().children).toBe('sopa de makako')
    });

});