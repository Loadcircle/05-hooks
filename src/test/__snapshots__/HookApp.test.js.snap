// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el hook app Match component 1`] = `
<div>
  sopa de makako
</div>
`;
